dark/views/tournament/add.py

```diff
@@ -11,8 +13,24 @@ class AddTournamentView(CreateView):
     form_class = AddTournamentForm
 
     def form_valid(self, form):
-        form.instance.creator = self.request.user
-        return super(AddTournamentView, self).form_valid(form)
+        creator = self.request.user
+
+        self.object = form.save(commit=False)
+        self.object.creator = creator
+        self.object.save()
+
+        tournament = Tournament.objects.get(pk=self.object.id)
+
+        tournament.participants.add(creator)
+
+        number_of_teams = form.cleaned_data.get('number_of_teams')
+
+        for i in range(number_of_teams):
+            team_name = 'Team ' + str(i + 1)
```

Comment:
na przyszłość (wrzuć do backloga) - te nazwy teamów powinno się dać ręcznie ustawić w formularzu podczas tworzenia

Reply:
Myślałem o tym, żeby po dołączeniu do zespołów użytkowniki będą mieć czas na wymyślenie i zupdatowania nazwy swojego zespołu. Dodatkowo organizator turneju będzie mógł zmieniać nazwy wszystkich zespołów jeśli będzie na to potrzeba.

Reply:
Zamierzam to zaimplementować w następnym sprint'ie